src/main/java/com/targomo/client/api/pojo/LocationProperties.java

```diff
@@ -0,0 +1,18 @@
+package com.targomo.client.api.pojo;
+
+import lombok.*;
+
+
+// Only relevant for POI service's gravitation requests
```

Comment:
If its only relevant for POI service requests why add it to the root class of locations. Would it make sense to rather create a new sub-class "PoiSourceCoordinate" or something?